Makefile

```diff
@@ -113,12 +118,26 @@ endif
 	find . -name \*~ -delete
 	find . -name \#\* -delete
 	rm -f bin/crio
+	rm -f bin/crio.cross.*
 	$(MAKE) -C conmon clean
 	$(MAKE) -C pause clean
 	rm -f test/bin2img/bin2img
 	rm -f test/copyimg/copyimg
 	rm -f test/checkseccomp/checkseccomp
 
+# the approach here, rather than this target depending on the build targets
+# directly, is such that each target should try to build regardless if it
+# fails. And return a non-zero exit if _any_ target fails.
+local-cross:
+	@$(MAKE) --keep-going $(CROSS_BUILD_TARGETS)
+
+bin/crio.cross.%: .gopathok
+	@echo "==> make $@"; \
+	TARGET="$*"; \
+	GOOS="$${TARGET%%.*}" \
+	GOARCH="$${TARGET##*.}" \
+	$(GO) build -i $(LDFLAGS) -tags "containers_image_openpgp" -o "$@" $(PROJECT)/cmd/crio
```

Comment:
I think you need `exclude_graphdriver_btrfs` and maybe `btrfs_noversion` in  tags.  Currently `bin/crio.cross.linux.amd64` is failing [with][1]:

```
vendor/github.com/containers/storage/drivers/btrfs/version.go:6:27: fatal error: btrfs/version.h: No such file or directory
 #include <btrfs/version.h>
```

I expect a working btrfs driver will be even harder on other OSes ;).

[1]: https://travis-ci.org/kubernetes-incubator/cri-o/jobs/388442795#L672

Reply:
Btrfs is missing on Linux because of #1599 (sorry :/), but we'll still need the tag(s) to exclude it for other OSes.

Reply:
We also don't want it on RHEL and Centos going forward.

Reply:
I still think we want tag(s) to exclude btrfs here.

Reply:
I guess we need to add btrfs_noversion
Wish these were positive rather then negative.   But @vbatts Adding this will eliminate attempts to build with btrfs.

Reply:
done